examples/step-50/step-50.cc

```diff
@@ -233,10 +243,10 @@ number Coefficient<dim>::average_value(
 template <int dim>
 template <typename number>
 std::shared_ptr<Table<2, VectorizedArray<number>>>
-Coefficient<dim>::create_coefficient_table(
+Coefficient<dim>::make_coefficient_table(
   const MatrixFree<dim, number, VectorizedArray<number>> &mf_storage) const
 {
-  auto coefficient_table =
+  std::shared_ptr<Table<2, VectorizedArray<number>>> coefficient_table =
```

Comment:
I prefer the ``auto`` here, as the type is redundant in this expression.

Reply:
But only if you know what `std::make_shared` returns. But who cares, I put the `auto` back in.